<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClubDataDialog</class>
 <widget class="QDialog" name="ClubDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="buddy">
        <cstring>clubNameEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="clubNameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Coach Name:</string>
       </property>
       <property name="buddy">
        <cstring>coachNameEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="coachNameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Country:</string>
       </property>
       <property name="buddy">
        <cstring>usaBtn</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="usaBtn">
       <property name="text">
        <string>USA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="canadaBtn">
       <property name="text">
        <string>Canada</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="otherCountryBtn">
       <property name="text">
        <string>Other</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="otherCountryEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Address 1:</string>
       </property>
       <property name="buddy">
        <cstring>address1Edit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="address1Edit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Address 2:</string>
       </property>
       <property name="buddy">
        <cstring>address2Edit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="address2Edit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>City:</string>
       </property>
       <property name="buddy">
        <cstring>cityEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cityEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>State:</string>
       </property>
       <property name="buddy">
        <cstring>stateCombo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="stateCombo"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Zip:</string>
       </property>
       <property name="buddy">
        <cstring>zipEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="zipEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ClubDataDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ClubDataDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
